src/icons/atm.svg

```diff
@@ -0,0 +1,79 @@
+<svg viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please add a `class="icon"` attribute to this `<svg>` element. 

Same comment for all other new SVG assets in this PR.